ui/.eslintrc.js

```diff
@@ -67,5 +66,7 @@ module.exports = {
     'react/no-unescaped-entities': 'off',
     'react/prop-types': 'off',
     'react/no-find-dom-node': 'off',
+    'jest/no-large-snapshots': ['error', {maxSize: 0}], // no shapshots please
```

Comment:
a bit heavy handed and reactionary

Reply:
i thought we all agreed no more snapshots.  linter rules make that automatic.